cli_tools/gce_ovf_import/daisy_utils/data_disk_workflow_updater_test.go

```diff
@@ -122,6 +87,29 @@ func TestAddDiskImportSteps(t *testing.T) {
 	assert.Equal(t, w.DefaultTimeout, w.Steps["wait-for-data-disk-2-signal"].Timeout)
 	assert.Equal(t, w.DefaultTimeout, w.Steps["delete-data-disk-2-import-instance"].Timeout)
 }
+func TestAddDiskImportStepsLongInstanceName(t *testing.T) {
+	w := createBaseImportWorkflow("a-very-long-instance-name-that-is-at-the-limit-of-allowed-leng")
+
+	diskInfos := []ovfutils.DiskInfo{
+		{FilePath: "gs://abucket/apath/disk1.vmdk", SizeInGB: 20},
+		{FilePath: "gs://abucket/apath/disk2.vmdk", SizeInGB: 1},
+	}
+	AddDiskImportSteps(w, diskInfos)
+
+	assert.NotNil(t, w)
+	assert.Equal(t, 2+4*len(diskInfos), len(w.Steps))
```

Comment:
Can you add a comment to explain what `2+4*len(diskInfos)` refers to?

Reply:
Added a comment